src/Plugin/Field/FieldFormatter/CaptureUtilityMapFormatter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\web_page_archive\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Field\FieldItemInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+
+/**
+ * Plugin implementation of the 'web_page_archive_capture_formatter' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "web_page_archive_capture_utility_map_formatter",
+ *   label = @Translation("Web page archive capture utility map formatter"),
+ *   field_types = {
+ *     "map"
+ *   }
+ * )
+ */
+class CaptureUtilityMapFormatter extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $elements = [];
+    foreach ($items as $delta => $item) {
+      $elements[$delta] = ['#markup' => $this->viewValue($item)];
```

Comment:
ummm, probably. I'm actually gonna create a separate issue on Drupal.org for overall caching concerns, so I may just defer that to then.  Does that work?